src/html/html_safety.rs

```diff
@@ -207,14 +207,4 @@ impl EscapeHtml for String {
             .replace("\"", "&quot;")
             .replace("'", "&#39;")
     }
-}
-
-impl EscapeHtml for &str {
```

Comment:
You did the opposite of what I suggested in here

Reply:
apologies for my blindness